src/server/api/routes/channels.router.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const channelsController = require('../controllers/channels.controller');
+
+/**
+ * @swagger
+ * /channels/{ID}:
+ *  delete:
+ *    summary: Delete a channel
+ *    description:
+ *      Will delete a channel with a given ID.
+ *    produces: application/json
+ *    parameters:
+ *      - in: path
+ *        name: ID
+ *        description: ID of the channel to delete.
+ *    responses:
+ *      200:
+ *        description: channel deleted
+ *      5XX:
+ *        description: Unexpected error.
+ *      400:
+ *        description: Bad Request.
+ */
+router.delete('/:id', (req, res) => {
+  channelsController
+    .deleteChannelById(req.params.id, req)
```

Comment:
```suggestion
    .deleteChannelById(req.params.id)
```

When the function is called deleteChannelById dont also add the req.